src/locales/de/InfoPage.json

```diff
@@ -15,22 +15,36 @@
   "Title": "Erläuterungen",
   "Background": {
     "Name": "Hintergrund",
-    "Paragraph1": "Für die Bewertung der Entwicklungstendenzen von Zoonosen und Zoonoseerregern sowie der Quellen von Erkrankungen des Menschen wird bundesweit einheitlich das Zoonosen-Monitoring gemäß Zoonosen-Stichprobenplan durchgeführt. Dies trägt dazu bei, die Belange der Richtlinie 2003/99/EG zu erfüllen. Übergreifendes Ziel ist, eine umfassende Bewertung der Entwicklungstendenzen von Zoonosen und Zoonoseerregern einschließlich Antibiotikaresistenzen sowie der Quellen von Erkrankungen des Menschen vornehmen zu können. Hierdurch soll auch die Durchführung wichtiger Komponenten der Richtlinie 2003/99/EG sowie des Durchführungsbeschlusses 2013/652/EU bzw. seiner Nachfolgeregelung 2020/1729/EU sichergestellt werden. ",
+    "Paragraph1": {
+      "Description1": "Für die Bewertung der Entwicklungstendenzen von Zoonosen und Zoonoseerregern sowie der Quellen von Erkrankungen des Menschen wird bundesweit einheitlich das ",
+      "BoldText": "Zoonosen-Monitoring",
```

Comment:
Reject: You loose the ordering information here.
Also, i recommend to not mix in the styling into the translation. So you can just keep the pattern descriptionX and let the views show it as bold. If you want to keep a hint in the translation you can use something like: `Description2_emphasized` or so.

Reply:
Done.